src/Routers/routes/ConsultationRoutes.tsx

```diff
@@ -6,6 +9,9 @@ import { AppRoutes } from "@/Routers/AppRouter";
 import { EncounterShow } from "@/pages/Encounters/EncounterShow";
 import { PrintPrescription } from "@/pages/Encounters/PrintPrescription";
 
+const ExcalidrawPage = lazy(() => import("@/components/Files/ExcalidrawPage"));
+const ExcalidrawView = lazy(() => import("@/components/Files/ExcalidrawView"));
```

Comment:
namings could be confusing, keep it more explicit

```suggestion
const ExcalidrawEditor = lazy(() => import("@/components/Files/ExcalidrawPage"));
const ExcalidrawView = lazy(() => import("@/components/Files/ExcalidrawView"));
```